README.md

```diff
@@ -160,6 +160,28 @@ using
 systemctl status earlyoom
 ```
 
+### Testing
+
+In order to see `earlyoom` in action, create/simulate a memory leak and let `earlyoom` do what it does: 
+
+```
+tail /dev/zero
+```
+
+### Checking Logs 
+
+If you need any further actions after a process is killed by `earlyoom` (such as sending emails), you can parse the logs by:
+
+```
+sudo journalctl -u earlyoom | grep -i kill
```

Comment:
this isn't listing everything for me.

`sudo journalctl -u earlyoom | grep -A1 sending` is what i'm using